src/client.ts

```diff
@@ -1055,6 +1055,16 @@ export class StreamChat<
     }
   }
 
+  _buildSort(
+    sort: UserSort | ChannelSort,
+  ): Array<{ field: string; direction?: AscDesc }> {
+    const sortFields: Array<{ field: string; direction?: AscDesc }> = [];
+    for (const [field, direction] of Object.entries(sort)) {
```

Comment:
There is no order guarantee here in es5 and in es6, insertion order to object (numeric/symbol are irrelevant to us) which should be documented probably.

Reply:
OK, I didn't know that. I will try to come up with a real solution then